DESIGN TEARDOWN PROJECT COMPLETED AND READY TO DEBUG LINTERS

angel-mora opened the pull request:
- HTML Validator checked
- Most of layout using grid, a little with flexbox
- Microverse 5th project

NOTE: (angel-mora) I have issues with my VSCODE so it looks like there's a third colaborator but it's just me.

angel-mora committed:
setting main files

Commit:
added basic planning in the readme file

me-skit committed:
Add main structure for the index.html

This commit adds the main structure of the index.html:
- main
  - header
  - article
  - footer

Commit:
completed html structure

Commit:
fix html structure and started css stardard settings for all browsers, about to set the grid layout

me-skit committed:
Put basic CSS structure

This commit add de basic CSS structure of the page

Commit:
started working on font and content position inside grids

me-skit committed:
Add some CSS structure for most of the sections and articles

This commit adds basic CSS structure for many of the articles and sections of
the project

Commit:
finished layout and start to color areas

me-skit committed:
Make the 'latest-title' overlap the sextions below

This commit just make a title overlap two sections below

me-skit committed:
Merge branch 'development' of github.com:angel-mora/smashing-mag into development

Commit:
added description to readme file,.sticler.yml and stylelint-config.js files

me-skit committed:
Adjust colors for some areas

This commit finish applying color for the main areas

angel-mora committed:
Merge branch 'development' of https://github.com/angel-mora/smashing-mag into development

angel-mora committed:
finish html debugging and ready to debug css inters

angel-mora committed:
fixed css indentation

angel-mora committed:
fixed extra warnings from linters

kobiyoyo reviewed (changes requested):
## Hi Team      ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
#### Great Job so far, but there are a few more things required. 
### General
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload
##### Navigation bar
* [x] Please consider using `div` in your navbar.
##### 3 articles rows
* [x] In this section it should contain 3 articles, each takes one-third of the width
Your's 
![1](https://user-images.githubusercontent.com/23278161/74992342-1d0d2280-5448-11ea-98fe-ed365af6ffa4.png)

The Original
![1](https://user-images.githubusercontent.com/23278161/74992371-3ca44b00-5448-11ea-979e-746638f0db55.png)

##### 1 article row
* [x] In this section it should contain 1 article, which takes the full width

### Make changes and submit another code review request.

*Happy coding*

me-skit committed:
Fix first review issues

This commit fix the next issues:
- Check HTML markup
- Enclose the navbar inside a div
- Arrange the section next the navbar to have 3 columns instead 4
- the las article next to the navbar now occupy the full width of the
  frame

euqueme commented on index.html:
```diff
@@ -0,0 +1,395 @@
+<body>
+    <main>
+        <header class="gray-header">
+            <div class="navbar">
```
[OPTIONAL] I strongly recommend changing the `<div> `tag in line #15 to be a `<nav>` tag

euqueme reviewed (approved):
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @angel-mora and @meme-es :balloon:  :lollipop: 

### Thank you for doing the changes requested by the previous reviewer :medal_sports: your project looks great and it's ready to be merged :heavy_check_mark: 

### I only have 3 requests for you but I've set them as optional

## Footer

- [ ] [OPTIONAL] I'm sorry this wasn't pointed out in the previous review but we actually require you to have `<a>` tags surrounding each footer link

## Live Demo

- [ ] [OPTIONAL] your Live Demo link takes me to your repo, consider fixing it or removing it from the Readme file

## OPTIONAL

- [ ] [OPTIONAL] I encourage the use of  the `<nav>` tag for the major navigation links, I have commented your code with a suggestion where you could use this tag :arrow_double_down: 

### That's it, team! :raised_hands: 

### Please do the changes (or not) and merge and submit a project completion form because it's approved :rocket: 

PROJECT STATUS: APPROVED :green_apple:
